### PR TITLE
Minor Update

### PR DESCRIPTION
**Pull request by FoggyMtnDrifter**
- Added Analytics to Site
- Switched Weblate URL in ReadMe badge

### CODE REVIEW

**Comment by hbjydev** [comment]
@ressonix can you update Prettier to ignore the src/i18n folder?
Also, make sure you `pre-commit run --all-files` quickly too to resolve those issues

File: gatsby-config.js [hunk 1]
```diff
@@ -122,5 +122,13 @@ module.exports = {
...
       },
     },
+    {
+      resolve: 'gatsby-plugin-matomo',
+      options: {
+        siteId: '1',
+        matomoUrl: 'https://analytics.rockylinux.org',
+        siteUrl: 'https://rockylinux.org'
+      }
+    },
```
Comment (hbjydev) [inline gatsby-config.js]: We can't deploy this live until we have a privacy policy. Waiting on @gmkurtzer for that one.

**Review by hbjydev — CHANGES_REQUESTED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (FoggyMtnDrifter) [reply gatsby-config.js]: That's fine, I think Neil is working on some things for Matomo today, anyways.

**Comment by FoggyMtnDrifter** [comment]
I'm just going to close this one for now until we get Matomo and the Privacy Policy sorted.

**Pull request closed without merging**